src/components/Buttons/EndCallButton/EndCallButton.tsx

```diff
@@ -20,10 +20,15 @@ const useStyles = makeStyles((theme: Theme) =>
 
 export default function EndCallButton(props: { className?: string }) {
   const classes = useStyles();
-  const { room } = useVideoContext();
+  const { room, removeProcessor } = useVideoContext();
+
+  const handleDisconnect = (): void => {
+    removeProcessor();
```

Comment:
I just realized the background is also applied in the prejoin screen when I refresh. I think instead of calling the remove processor here, maybe we should call it in the prejoin screen.